server/bin/www

```diff
@@ -3,6 +3,23 @@
 /* Sets up the environment variables from your .env file*/
 require("dotenv").config();
 
+/* Sets up the PostgreSQL database connection. */
+const { Sequelize } = require('sequelize');
+const sequelize = new Sequelize('postgres', 'postgres', '11111111', {
```

Comment:
For sequelize instantiation, can it be possible to add support for environment configs as well? The main reason is because I have a different password set for pg so the password provided does not work for me. I was thinking of something along the lines of:
```
const sequelize = new Sequelize(
  'postgres', 
  'postgres', 
  process.env.DB_PASSWORD || '11111111', { // I can provide my own db password without changing credentials you provided
  host: 'localhost',          
  dialect: 'postgres'
});
```